scripts/opt/backup-loop.sh

```diff
@@ -439,7 +439,9 @@ restic() {
       log ERROR "RESTIC_REPOSITORY is not set!"
       return 1
     fi
-    if output="$(command restic cat config 2>&1 >/dev/null)"; then
+
+    exec 5>&1
+    if output="$(command restic cat config 2>&1 >/dev/null | stdbuf -oL awk '{print "restic cat config: " $0}' | tee >(cat - >&5))"; then
```

Comment:
It would be great if you could add an inline comment that explains what each step of the pipeline is doing. This is where bash gets powerful but cryptic 😄